documentation/docs/liquidity-mining/upcoming/DAI_on_Bamboo.md

```diff
@@ -0,0 +1,43 @@
+## Bamboo Relay
+[Bamboo Relay](https://bamboorelay.com/) is a decentralized crypto exchange built upon 0x protocol. Users can trade, lend or borrow any ERC20 token trustlessly from their own wallet. [0x](https://0x.org/) sponsors this program with a goal to improve the liquidity for its Relayers.   
+
+## Timeline
+
+This liquidity mining program will be initially running for 12 weeks from early February 2020 with probability for extension. 
```

Comment:
This liquidity mining program will run for 12 weeks starting in early February 2020.

(I think we can update re: extension once there's more certainty)

Reply:
(change throughout)